source/routing/tests/routing_api_test.cpp

```diff
@@ -2368,6 +2368,290 @@ BOOST_AUTO_TEST_CASE(section_geometry_without_shapes) {
+ *            section 1 -> crow fly with real duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *
+ * Case 2 :
+ * - Type : With a big from filter radius
+ * - Result : We have 4 journeys with 3 sections each
+ *            for each journeys
+ *            section 1 -> crow fly with 0 duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *
+ * Case 3 :
+ * - Type : With a half from filter radius
+ * - Result : We have 4 journeys with 3 sections each
+ *            * jouneys 1
+ *            section 1 -> crow fly with 0 duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *           * jouneys 2
+ *            section 1 -> crow fly with 0 duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *           * jouneys 3
+ *            section 1 -> crow fly with real duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *            * jouneys 4
+ *            section 1 -> crow fly with real duration
+ *            section 2 -> public transport
+ *            section 3 -> crow fly with real duration
+ *
+ *
+ *
+ *                       SA1                          SA2
+ *             |-------------------------|---------------------------|
+ *             |                         |                           |
+ *             |                         |                           |
+ * x from      | S1             S2       |       S1             S2   |
+ *   ------------  -------------  --------------- --------------     |
+ *       10 m             10 m            10 m           10 m        |
+ *             |                         |                           |
+ *             |                         |                           |
+ *             |                         |                           |
+ *             |-------------------------|---------------------------|
+ *
+ *
+ *
+ *
+ *                                    SA3
+ *             |--------------------------------------------------|
+ *             |                                                  |
+ *             |                                                  |
+ *             |                                                  |
+ * x to        |  S1          S2           S3          S4         |
+ *   -------------  ----------  -----------  -----------          |
+ *        100 m        100 m       100 m       100 m              |
+ *             |                                                  |
+ *             |                                                  |
+ *             |                                                  |
+ *             |--------------------------------------------------|
+ *
+ */
+BOOST_AUTO_TEST_CASE(journeys_with_free_radius_filter) {
```

Comment:
it would be nice if we have some tests on odt and admins, either in cpp  or jormun